demo-app/src/component.js

```diff
@@ -14,10 +14,9 @@ window.onhashchange = function() {
 let iframeClient = new Client();
 iframeClient.start();
 
-iframeClient.onPubsub(publication => {
+iframeClient.messaging.addListener("host.topic", publication => {
```

Comment:
I used messaging instead of pubSub, just because I think its friendlier... Open for feedback